create_dmd_release/create_dmd_release.d

```diff
@@ -471,13 +468,13 @@ void buildAll(Bits bits, string branch, bool dmdOnly=false)
     {
         info("Building Druntime 32mscoff");
         changeDir(cloneDir~"/druntime");
-        run(msvcVarsX86~makecmd~msvcEnv~" druntime32mscoff");
+        run(msvcVars~makecmd~msvcEnv~" druntime32mscoff");
```

Comment:
You have to switch to the x86 environment here to get the correct compiler for -m32mscoff.

Reply:
I think that's done by `msvcVars` which activates either x86 or x64, see https://github.com/dlang/installer/pull/441/files#diff-5c352fb76ca5590e7ad3d2640c695272R385.

Reply:
I see, that's a 32-bit build in the 64-bit branch, weird that it worked.

Reply:
I think it will compile and create a library, but it will have mixed architecture in it (x64 object files from cl.exe and 32-bit files from dmd).

Reply:
Fixed now.